oauth2_http/javatests/com/google/auth/oauth2/UserCredentialsTest.java

```diff
@@ -830,6 +830,27 @@ public void IdTokenCredentials_NoUserEmailScope_throws() throws IOException {
     }
   }
 
+  @Test
+  public void toBuilder() throws IOException {
```

Comment:
Please rename similarly to other tests, so the name describes what is being tested and expected behavior. 
Like UserCredentians_ToBuilder_AccessTokenSaved